torch/jit/__init__.py

```diff
@@ -275,6 +275,17 @@ def _unique_state_dict(module, keep_vars=False):
     return filtered_dict
 
 
+def _remove_unused_state_batchnorm(module, state_dict):
```

Comment:
Looks to me this is onnx specific, could you move this to torch/onnx?

Reply:
This is necessary in LegacyTracedModule as well. There are unused keys in the state dict, which should be removed.

Reply:
@bddppq

Reply:
@bddppq please review the updates